web/client-api/src/main/java/io/deephaven/web/public/linked/index.html

```diff
@@ -1,184 +1,225 @@
+      through the MessagePort. The "target" window, listening for incoming
+      messages from that MessagePort, will respond by taking the selected value
+      and replacing its current filter to match the selected row in the "source"
+      window.
+    </p>
+    <h3>Links for further reading</h3>
+    <ul>
+      <li>
+        <a
+          href="https://developer.mozilla.org/en-US/docs/Web/API/HTML_Drag_and_Drop_API"
+          >Drag and Drop API</a
+        >, supporting passing messages between browser windows that are not
+        aware of each other's existence or current state.
+      </li>
+      <li>
+        <a
+          href="https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage"
+          >Sending messages to another window</a
+        >
+        - pay special attention to the "Security Concerns" section to make sure
+        messages can't be misused.
+      </li>
+      <li>
+        <a
+          href="https://developer.mozilla.org/en-US/docs/Web/API/MessageChannel"
+          >MessageChannel</a
+        >
+        and
+        <a href="https://developer.mozilla.org/en-US/docs/Web/API/MessagePort"
+          >MessagePort</a
+        >
+        APIs, allowing two JS contexts (like windows or workers) which otherwise
+        can't communicate be "introduced" through another window
+      </li>
+      <li>
+        <a
+          href="https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Transferable_objects"
+          >The Transferable interface</a
+        >, present on types which can be sent via (Window or MessagePort)
+        postMessage, allowing certain objects to be moved rather than copied.
+      </li>
+    </ul>
+
+    <h3>Persistent Query Configurations</h3>
+    <p>
+      Selecting a table will attempt to open it in a new window - to make this
+      demo easier to use, resize the windows so that more than one table is
+      visible at once.
+    </p>
+    <ul id="queries"></ul>
+    <script type="module">
+      import dh from "./dh-core.js";
+
+      const { Client } = dh;
+      var url = new URL("/socket", window.location);
+      if (url.protocol === "http:") {
+        url.protocol = "ws:";
+      } else {
+        url.protocol = "wss:";
       }
-    }
-    function addQuery(queryInfo) {
-      console.log("config added: " + queryInfo.name);
 
-      var queryElt = document.createElement('li');
-      configs[queryInfo.serial] = {info:queryInfo, elt:queryElt};
-      queryElt.innerText = queryInfo.name + ' [' + queryInfo.status + ']';
-      var tables = document.createElement('ul');
-      queryElt.appendChild(tables);
-      queryInfo.tables.forEach(t => {
-        var tableElt = document.createElement('li');
-        tableElt.innerText = t;
-        tables.appendChild(tableElt);
-        tableElt.onclick = e => {
-          e.stopPropagation();
-          openTable(queryInfo.name, t);
-        };
+      window.c = new Client(url.href);
```

Comment:
this is still DHE - the whole example is not ready for be shared, so while we have it in the dhc codebase, it isnt linked from the index.html up a directory.

we can delete it if you'd prefer, or take the time to rebuild it to work with dh.CoreClient instead of Client (which doesnt exist)?